model/src/main/kotlin/licenses/ResolvedLicenseFileInfo.kt

```diff
@@ -63,4 +63,9 @@ data class ResolvedLicenseFile(
      * The unarchived license file.
      */
     val file: File
-)
+) {
+    /**
+     * Returns the content of the [license file][file].
```

Comment:
How about just "Return" for consistency?

Reply:
Done.